_pytest/test_utf8_helpers.py

```diff
@@ -35,7 +35,11 @@ def test_encode_preserves_string_without_utf8():
     assert b'test' == encode_to_utf8(u'test')
 
 def test_encode_preserves_byte_strings():
-    assert b'æøå' == encode_to_utf8(b'æøå')
+    try:
+        unicode        # Python 2
+        assert b'æøå' == encode_to_utf8(b'æøå')
```

Comment:
This causes a syntax error in Python 3 ("bytes can only contain ASCII literal characters"), so we need a different fix than this.